docs/Maths.md

```diff
@@ -226,6 +226,107 @@ Result:
 ![](plotly_svg.png)
 
 
+## Using Pygal ##
+[Pygal](https://www.pygal.org/en/stable/) is a graph plotting library using python. You can install Pygal using ` $ pip install pygal` command. 
+
+`fpdf2` is able to embed the graph and charts that are generated using `Pygal` library. The following ways explain how to embed `Pygal` charts into `fpdf2` library.
+
+### Using cairosvg (*A faster and efficient implementation*) ### 
```

Comment:
Your examples convert the SVG output of pygal to a raster file (PNG) before passing it to fpdf2, even with two different methods and three seperate third-party libraries.
Why?
The resulting PDF file will be much larger and the diagram can't be scaled losslessly anymore. For the user of the produced file this is very inefficient and causes a severe loss of quality.

It would make a lot more sense to just show how to feed the SVG from PyGal directly to fpdf2. After all, this capability is one of the key features that distinguishes fpdf2 from other PDF libraries.

A comparison of how otherwise unrelated libraries can convert SVG data to raster images does not really seem to be a relevant topic for the fpdf2 documentation.

Reply:
@gmischler this was already discussed in https://github.com/PyFPDF/fpdf2/issues/652#issuecomment-1501601977

Have you read the discussion in this issue? 😊

Reply:
Ah I see. That's a rather annoying way to create SVGs (even referencing online fonts). I'm sure it makes it much simpler for them, though... :wink:

In that case, maybe the example should actually mention that the SVG files generated by PyGal can't be used directly because of the special features they use. The current phrasing makes the reader think that the files could be imported directly, but then without explanation does it in a different way.

Thinking about it some more, the general approach shown is not really specific to PyGal. Maybe this information should rather be titled "Importing Graphics from Incompatible SVG Data", with PyGal used as an example? It could even go to the "SVG.md" page, because it doesn't only apply to graphs.

Reply:
> In that case, maybe the example should actually mention that the SVG files generated by PyGal can't be used directly because of the special features they use. The current phrasing makes the reader think that the files could be imported directly, but then without explanation does it in a different way.

That's exactly what I suggested: https://github.com/PyFPDF/fpdf2/pull/762#discussion_r1162460377

Reply:
Hello @gmischler , thank you so much for reviewing this PR and sharing valuable insights. 
I tried to do some research on my own and was not clear about the reason behind not being able to import incompatible SVG data into `fpdf2` until @Lucas-C shared it with me. 
I will try to reflect on the suggestions in my next commit. 
And yes,
> After all, this capability is one of the key features that distinguishes fpdf2 from other PDF libraries

I am aligned with the same thought. Being a beginner in open-source contribution, I believe I am learning every single day and I wish to become an active contributor like you people. 

@Lucas-C I will try to follow the advice as much as I can.